client/src/app/shared/shared-main/misc/top-menu-dropdown.component.ts

```diff
@@ -9,11 +9,12 @@ import { NgbDropdown, NgbModal } from '@ng-bootstrap/ng-bootstrap'
 export type TopMenuDropdownParam = {
   label: string
   routerLink?: string
+  isHidden?: boolean
```

Comment:
I think it should be a method so we can put a dynamic value here. (not needed now, but can be useful in the future)

Reply:
Ok I understood what you mean =)